<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Search</class>
 <widget class="QDialog" name="Dialog_Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Firmware</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rrcc.qrc">
    <normaloff>:/png/png/internet.png</normaloff>:/png/png/internet.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Search Parameter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="toolTip">
           <string>Enter Filename (???? will be replaced with version)</string>
          </property>
          <property name="text">
           <string notr="true">v11_00????.fullos.pkg</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_server">
          <property name="toolTip">
           <string>Select Server</string>
          </property>
          <item>
           <property name="text">
            <string notr="true">cdn.awsbj0.fds.api.mi-img.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">cdn.awsde0.fds.api.mi-img.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">cdn.cnbj0.fds.api.mi-img.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">cdn.cnbj2.fds.api.mi-img.com</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox_model">
          <property name="toolTip">
           <string>Select Model</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Generation 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Generation 2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="spinBox_Start">
            <property name="toolTip">
             <string>Start Version</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_Stop">
            <property name="toolTip">
             <string>Stop Version</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="toolTip">
         <string>Search Progress</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string notr="true">0000 | 0 / 0 | %p%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="toolTip">
         <string>Doubleclick to download selected firmware</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>0</number>
        </attribute>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Size</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Firmware</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <item>
         <property name="text">
          <string notr="true"> 0000-00-00 </string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="text">
          <string notr="true"> 00:00:00 </string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="text">
          <string notr="true"> 000000000 </string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="text">
          <string notr="true"> v11_000000.fullos.pkg </string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_model</tabstop>
  <tabstop>comboBox_server</tabstop>
  <tabstop>spinBox_Start</tabstop>
  <tabstop>spinBox_Stop</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>treeWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../rrcc.qrc"/>
 </resources>
 <connections/>
</ui>
